quickjs-libc.c

```diff
@@ -1321,7 +1321,7 @@ static JSValue js_std_file_tell(JSContext *ctx, JSValueConst this_val,
     int64_t pos;
     if (!f)
         return JS_EXCEPTION;
-#if defined(__linux__)
+#if defined(__linux__) || defined(__GLIBC__)
```

Comment:
Ditto, musl implements it: https://github.com/kraj/musl/blob/9389b23dd492040084a2f925f815c35d036f7e4d/include/stdio.h#L144